src/app/components/normal-component/pick-game/pick-game.component.ts

```diff
@@ -134,6 +137,8 @@ class GameDescription {
 
     public static readonly GO: Localized = () => $localize`The oldest strategy game still practiced widely. A territory control game.`;
 
+    public static readonly HIVE: Localized = () => $localize`You are in charge of a hive full of insects. Use your insects' abilities to block the opponent's queen in order to win!`;
```

Comment:
insects' abilities ou insect's abilities ?

Reply:
insects' car "les abilités de vos insectes"
Mais tu préfères peut-être "Use the abilities of your insects to block ..." ?

Reply:
oui j'avoue, c'est p-e officiel mais c'est chelou l'apostrophe à la fin comme ça :D

Reply:
quoi tu insinue que y'a des constructions louches dans certaines langues ? :D
insects's serait encore plus chelou ! 
Tu préfère : Use the abilities of your insects to ... ?

Reply:
oui je péfère !

Reply:
À gris !